convert2rhel/subscription.py

```diff
@@ -390,3 +390,44 @@ def rollback():
         unregister_system()
     except OSError:
         loggerinst.warn("subscription-manager not installed, skipping")
+
+
+def download_subscription_manager_packages():
+    from convert2rhel.systeminfo import system_info
+    from convert2rhel.utils import store_content_to_file
+    from convert2rhel.pkghandler import call_yum_cmd
+
+    _CENTOS_CONTRIB_REPO = '[contrib]\n' \
+                           'name=CentOS-6.10 - Contrib\n' \
+                           '#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=contrib\n' \
+                           'baseurl=http://mirror.centos.org/centos/6.10/contrib/$basearch/\n' \
+                           'gpgcheck=1\n' \
+                           'enabled=1\n' \
+                           'gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6'
+
+    _TEMP_REPOFILE_DIR = '/tmp/convert2rhel/'
+    _TEMP_REPOFILE = _TEMP_REPOFILE_DIR + 'centos_contrib.repo'
+
+    version_full_number = re.search(r"([\d.]+)", system_info.system_release_file_content).group(1)
+    version_major = re.search(r"^(\d+)?\.?(\d+)?\.?(\*|\d+)?$", version_full_number).group(1)
+    version_minor = re.search(r"^(\d+)?\.?(\d+)?\.?(\*|\d+)?$", version_full_number).group(2)
+
+    if int(version_major) <= 6 and int(version_minor) <= 6:
```

Comment:
In this `if` block you check that `int(version_major) <= 6` and then in the line 420 you check that  `int(version_major) == 6`. So it is possible that version is 6 and `if` statement on line 420 could be skipped.
I'd use if-else here

Reply:
Actually, the first condition is to check if version is = or < 6.6, because we need to make sure to run only on 6.7 or above